CHANGELOG.md

```diff
@@ -1,15 +1,128 @@
 # Changelog
 
+## [0.57]
+
+Welcome to the 0.57 release of React Native! This release addresses a number of issues and has some exciting improvements. We've intentionally left this release in an extended release candidate state in order to improve quality. 
+
+You may notice that this release's changelog is thinner than previous versions; that's not from lack of hard work -- this release includes 566 commits by 78 different contributors! Instead, we've taken feedback and prepared a changelog that contains only user impacting changes. Please share your feedback and let us know how we can make this even more useful.
+
+### Highlights
+
+- Accessibility APIs now support accessibility hints, inverted colors, and easier usage of defining the element's role and states; read more at [@ziqichen6's excellent blog post](https://facebook.github.io/react-native/blog/2018/08/13/react-native-accessibility-updates)
+- Android is now using SDK 27, gradle 3.1, and support library 27.x; building with Android plugin 3.2 doesn't work due to the gradle scripts, so please stay on Android Studio 3.1 for now
+- We've changed the dependency tree around, specifically around babel presets -- if you have a custom `.babelrc` in place make sure you update it after upgrading
+- There is now built-in TypeScript support in the Metro bundler
```

Comment:
I think you can expand this a bit using this https://github.com/facebook/react-native/issues/20030#issuecomment-410705553, or just link it for reference

Reply:
We could also link this tbh https://blogs.msdn.microsoft.com/typescript/2018/08/27/typescript-and-babel-7/